composer.json

```diff
@@ -23,8 +23,9 @@
     "php" : "^7.0",
     "illuminate/contracts": "~5.1.0|~5.2.0|~5.3.0",
     "illuminate/support": "~5.1.0|~5.2.0|~5.3.0",
-    "google/apiclient" : "^1.1",
-    "nesbot/carbon": "^1.21"
+    "google/apiclient" : "^2.0",
+    "nesbot/carbon": "^1.21",
+    "tedivm/stash": "^0.14.1"
```

Comment:
@freekmurze from what I've seen, it is possible to use laravel's caching capabilities, but we would have to create a bridging between psr-6 and https://github.com/illuminate/cache

Do you think we should move in that direction ?